src/main/scala/org/ergoplatform/local/TransactionGenerator.scala

```diff
@@ -88,13 +88,13 @@ class TransactionGenerator(viewHolder: ActorRef,
     val payloadReq: Future[Option[TransactionRequest]] = wallet.confirmedBalances().map { balances =>
       Random.nextInt(100) match {
         case i if i < 70 =>
-          Some(PaymentRequest(randProposition, randAmount, None, None))
+          Some(PaymentRequest(randProposition, math.min(randAmount, balances.balance), None, None))
```

Comment:
I think `balances.balance - feeReq.value` will be correct here

Reply:
@catena2w, fixed.